cli.py

```diff
@@ -67,31 +67,30 @@ async def inspect_many_blocks_command(
     max_concurrency: int,
     request_timeout: int,
 ):
-    inspect_db_session = get_inspect_session()
-    trace_db_session = get_trace_session()
-
     inspector = MEVInspector(
         rpc,
-        inspect_db_session,
-        trace_db_session,
         max_concurrency=max_concurrency,
         request_timeout=request_timeout,
     )
+    inspect_session, trace_session = get_sessions()
     await inspector.inspect_many_blocks(
-        after_block=after_block, before_block=before_block
+        after_block=after_block,
+        before_block=before_block,
+        inspect_session=inspect_session,
+        trace_session=trace_session,
     )
 
 
 @cli.command()
 @coro
 async def fetch_all_prices():
-    inspect_db_session = get_inspect_session()
+    inspect_session, _ = get_sessions()
```

Comment:
Good point, otherwise the option would be making these return session makers instead of actual sessions, and refactor the code downstream

Reply:
I think make the session instead of returning sessionmaker is kinda nice. I'd imagine we'd just call the sessionmaker immediately most of the time otherwise.